eng/Versions.props

```diff
@@ -200,7 +200,7 @@
     <XamarinMacOSWorkloadManifestVersion>12.1.301-preview.13.2</XamarinMacOSWorkloadManifestVersion>
     <XamarinTvOSWorkloadManifestVersion>15.2.301-preview.13.2</XamarinTvOSWorkloadManifestVersion>
     <MonoWorkloadManifestVersion>$(MicrosoftNETCoreAppRefPackageVersion)</MonoWorkloadManifestVersion>
-    <MicrosoftNETWorkloadEmscriptenManifest70100Version>7.0.0-preview.3.22122.1</MicrosoftNETWorkloadEmscriptenManifest70100Version>
+    <MicrosoftNETWorkloadEmscriptenManifest70100Version>7.0.0-preview.3.22128.1</MicrosoftNETWorkloadEmscriptenManifest70100Version>
```

Comment:
@lewing @mmitche looks like this version is out of sync somehow. looks like it should be 22121.1 based on this PR: https://github.com/dotnet/runtime/pull/63894/files  Any ideas why it didn't get updated?
```
Exception: "Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadManifestCompositionException: Workload manifest dependency 'Microsoft.NET.Workload.Emscripten' version '7.0.0-preview.3.22128.1' is lower than version '7.0.0-preview.3.22167.1' required by manifest 'microsoft.net.workload.mono.toolchain' [D:\a\_work\1\s\artifacts\bin\redist\Release\dotnet\sdk-
```

Reply:
It's not coherency tied to runtime: https://github.com/dotnet/installer/pull/13376/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R169

@wtgodbe removed it here: https://github.com/dotnet/installer/pull/13376/commits/f92d2f6b3248ac5b729405a1e2829a73d9bb64b0

Reply:
As I understand https://github.com/dotnet/installer/issues/13450 related to this?

Reply:
@kirsan31 I don't think that's related.

Reply:
#13450 is related in that current installer builds have a 2 week old runtime. Not sure if we have a way to prevent this sort of incoherency (short of repo merge) as runtime always appears to be the slowest.

As for emscripten, I think we wanted that removed downlevel but not here.

Reply:
runtime should be updated in sdk more often than strictly once a day. everything else updates like 20 times a day so this (unwritten) rule does not make any sense.

emscripten commit from https://github.com/dotnet/runtime/pull/63894 is still not available in this pr.

it takes 5-6 hours to merge sdk pr and everytime involves human to manually re run the job https://github.com/dotnet/sdk/pull/24504, this is slowing things down, there is no upside

Reply:
merging installer and sdk sounds great. it will simplify things 👍

Reply:
> runtime should be updated in sdk more often than strictly once a day

nit: dotnet/runtime only does official builds twice a day and those fail a fair amount. Dependencies flowing from there once per day sounds pretty good considering.

Reply:
Runtime does normal batched official builds now. 7 builds yesterday.

Reply:
@mmitche daily build table wasn't updated https://aka.ms/dotnet/7.0.1xx/daily/productCommit-linux-x64.txt shows the same (22 days old) runtime commit hash. is it expected?

Reply:
Official build is just finishing up now.

Reply:
latest bits are available now. thanks!

Reply:
> once per day sounds pretty good considering.

i am not sure why is that a good thing? the sooner we know that upcoming version of runtime has a problem or is all green, the better. when there is a problem, the automatic pr will be red, people will take a look when the have time next.

currently, you have to wait for the whole day, then dotnet/sdk pr updating dotnet/runtime always takes 5-10 commits behind the head at the time when it opens the pr, then it takes another half a day to get the final installer (assuming everything was green thereafter).

if this defensive strategy is not really _buying_ us something and only postponing bug reports by a day, let the dotnet/runtime dependency flow as many times as the batched official build flows... for the 99% of the times when everything _is_ green.